make-release.sh

```diff
@@ -28,6 +28,32 @@ usage ()
   exit 1
 }
 
+checkForBlockerIssues()
+{
+    # check for blockers only if doing a 7.yy.0 release
+    if [[ ${CHE_VERSION} == *".0" ]]; then 
+        # install gh cli from https://github.com/cli/cli/releases/
```

Comment:
I would not use cli for such simple request but anyway this cli is already installed on runners